### PR TITLE
Colored Filing Cabinets(also renameable)

### PR DESCRIPTION
**Pull request by redmoogle**
# General Documentation

### Intent of your Pull Request

Add craftable colored variants of the average filing cabinet

Can be renamed with a pen.
Can be recolored with a spraycan

image

### Why is this change good for the game?

Department colored cabinets and also _craftable_

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
You can craft a cabinet for 2 metal, taking 3 seconds each

### What general grouping does this PR fall under? 
Engineering changes?

# Changelog
:cl:  
rscadd: Craftable colored cabinets
/:cl:

### CODE REVIEW

**Comment by redmoogle** [comment]
~~uhh don't merge until either I get this refactor or just give up on the refactor~~ Finished

**Comment by Sniblet** [comment]
Haha yes the new ultimate clockcult cheese

File: code/modules/paperwork/filingcabinet.dm [hunk 1]
```diff
@@ -17,6 +17,7 @@
 	icon_state = "filingcabinet"
 	density = TRUE
 	anchored = TRUE
+	var/colour = null
```
Comment (Bibby0110) [inline code/modules/paperwork/filingcabinet.dm]: autodoc

**Review by Bibby0110 — COMMENTED** [review]
autodoc moment

**Review by JamieD1 — APPROVED** [review]
